recipes/gobgen

```diff
@@ -0,0 +1,3 @@
+(gobgen :fetcher github
+        :repo "gergelypolonkai/gobgen.el"
+        :files ("gobgen.el"))
```

Comment:
Please don't specify `:files` attribute. In this case, it is enough to use default value(`:defaults`). Please read https://github.com/melpa/melpa#recipe-format .